src/AnonParamsSelectionStep/AnonParamsSelectionStep.tsx

```diff
@@ -33,7 +33,7 @@ export const AnonParamsSelectionStep: FunctionComponent<AnonParamsSelectionStepP
     <>
       <div className="AnonParamsSelectionStep notebook-step">
         <NotebookNavAnchor step={NotebookNavStep.AnonParamsSelection} status="done" />
-        <Title level={3}>Adjust suppression threshold</Title>
+        <Title level={3}>Anonymization parameters</Title>
```

Comment:
Paul made a related change today:

https://github.com/diffix/desktop/blob/bbe74a7e33e923d884ba396ae43d7f7d6800baa2/src/Notebook/notebook-nav.tsx#L286-L290

Should these be in sync?

Reply:
Ah, missed that. I will make the section heading similar to the step title.